### ISSUE TITLE
Issues encountered while parsing CIF

### ISSUE THREAD

**Issue by YuanbinLiu** [arosen93/QMOF#19]
Dear authors,

When I run main.py, I got a UserWarning "Issues encountered while parsing CIF: Some fractional co-ordinates rounded to ideal values to avoid issues with finite precision". Do this warning have any effect on the results of CGCNN?

**Comment by arosen93** [comment]
Hi @YuanbinLiu. 

The warning you received is from [Pymatgen](https://github.com/materialsproject/pymatgen). Specifically, the warning is from [this section](https://github.com/materialsproject/pymatgen/blob/77c6dd669e159ca5fc5b2605b5a07b0bd3e500f9/pymatgen/io/cif.py#L552) of Pymatgen's CIF parser. In short, this should have no noticeable effect on the CGCNN training or evaluation. The rounding, should it occur, is extraordinarily small.

**Comment by YuanbinLiu** [comment]
Got it, thanks a lot.

**Comment by SANTKJD** [comment]
Dear authors，

When I run main.py，I got a
warning "warnings.warn("Issues encountered while parsing CIF: " + "\n".join(self.warnings))".After running I just got two IDs in test_results.csv.Can you tell how to deal with it?

**Comment by arosen93** [comment]
The Pymatgen-generated warning mentioned above is not an issue. If you are only receiving two entries in your testing set, have you made sure to download the dataset of 20k+ MOFs? By default, this repo only ships with a toy example. You need to download the data from Figshare.

**Comment by SANTKJD** [comment]
I have trained 29000+ cif files

**Comment by arosen93** [comment]
Without further details, I can't help debug your issue for you.

I should mention that the CGCNN code used here is almost identical to the
original (https://github.com/txie-93/cgcnn). If you have issues with CGCNN,
you may wish to consider posting an issue there.

**Comment by SANTKJD** [comment]
![image](https://user-images.githubusercontent.com/105814587/178157369-5ea5c7a8-28b5-4a9c-9751-da1687ddd2e2.png)

![image](https://user-images.githubusercontent.com/105814587/178157399-74cdb7e7-a09a-442b-9971-7c69821e20c3.png)

**Comment by SANTKJD** [comment]
Here is the test_results.
![image](https://user-images.githubusercontent.com/105814587/178157470-08f5909a-1c17-44b2-9be6-5cd46cbf9f0f.png)

**Comment by arosen93** [comment]
According to your image above, you specifically requested a test set size of 2. As you requested, you can see there are only two entries in the testing set CSV.

**Comment by SANTKJD** [comment]
But the value I got seems  much different from the actual value.

**Comment by arosen93** [comment]
Your training set size is only 8. That's orders of magnitude smaller than what you need for deep learning. It's no surprise your testing set results are poor. This is not an issue with the code, so please direct your more fundamental comments to either the parent CGCNN repo or a machine learning forum.

**Comment by SANTKJD** [comment]
OK，I will try again，thanks a lot .

**Comment by SANTKJD** [comment]
![image](https://user-images.githubusercontent.com/105814587/178168796-dd3d80b2-258f-4a3b-b8cb-6bfb93da5dc2.png)
![image](https://user-images.githubusercontent.com/105814587/178168854-63ed3e0f-f061-4e8e-bea4-c534ba89423d.png)
![image](https://user-images.githubusercontent.com/105814587/178168966-466a6f6f-4008-4aaf-9f84-cb7732c16810.png)
Dear author，
I found that the number I set affects whether it reports errors or not.
